### PR TITLE
Cleanup/Refactor

### PR DESCRIPTION
**Pull request by divergentdave**
Hi! I'm responding to your call on Twitter. I'll be cleaning up style issues, adding some idioms, and maybe even some more object-orientedness in this PR.

### CODE REVIEW

**Comment by divergentdave** [comment]
Alright, I'd say this is now sufficiently over-engineered :stuck_out_tongue_winking_eye:

File: drawframe.py [hunk 1]
```diff
@@ -74,12 +74,10 @@ def __init__(self, x, y):
 
 
 def place_scenery(canvas, y):
-    x = 0
-    for block in range(int(canvas.width)):
+    for x in range(0, canvas.width, BLOCK_SIDE):
```
Comment (divergentdave) [inline drawframe.py]: FYI this was looping too many times. At the end, x would be SCREEN_WIDTH \* BLOCK_SIDE.

**Comment by thisisparker** [comment]
Thank you so much for this! It took me a while to review it but when I finally got a chance to look it was INCREDIBLY helpful—this was like a session in a classroom except I was learning the whole dang time. The only part that hasn't fully clicked for me is some of the stuff around your refactoring of the classes, but I will spend a little more time with it.

I really, greatly appreciate this. Owe you big time.

**Comment by divergentdave** [comment]
You're welcome, I'm glad you found it helpful! Let me know if you have any questions about the class stuff, I'd be happy to explain more.

**Pull request merged**